packages/blockchain-wallet-v4-frontend/src/data/goals/sagas.js

```diff
@@ -516,11 +512,16 @@ export default ({ api }) => {
       return yield put(actions.modals.showModal(kycDocResubmit.name))
     }
     if (sunriver) {
-      return yield put(actions.modals.showModal(sunriver.name, sunriver.data))
+      // return yield put(actions.modals.showModal(sunriver.name, sunriver.data))
```

Comment:
Should we just delete `sunriver` block entirely? at least probably want to `return`?